### PR TITLE
Database

### PR DESCRIPTION
**Pull request by GuillermoRosales**
The scripts have more than one image, so i modified the model to have
that feature, and i improve the style of my code.

### CODE REVIEW

**Comment by nicolaquartieri** [comment]
Guille your PR is not able to merge. Please review this.

**Comment by GuillermoRosales** [comment]
Ok Nicolas, i think i solved it, this is the first time that i make something like this, so i hope that is Ok, im pending of your comments.

File: scriptsAPADEA.iml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (nicolaquartieri) [inline scriptsAPADEA.iml:1]: Why are you committing this file ?

File: .idea/gradle.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (nicolaquartieri) [inline .idea/gradle.xml:1]: Why are you committing this file ?

Comment (GuillermoRosales) [inline .idea/gradle.xml:1]: It supposed to be in.git ignore isn't it?

Comment (nicolaquartieri) [inline .idea/gradle.xml:1]: This file is already in DEV. Check the difference between DEV and this branch and you will see.

**Pull request closed without merging**